public/app/features/explore/state/utils.ts

```diff
@@ -67,10 +67,18 @@ export const makeExplorePaneState = (): ExploreItemState => ({
   panelsState: {},
 });
 
-export const createEmptyQueryResponse = (): PanelData => ({
+export const createEmptyQueryResponse = (): ExplorePanelData => ({
   state: LoadingState.NotStarted,
   series: [],
   timeRange: getDefaultTimeRange(),
+  graphFrames: [],
+  logsFrames: [],
+  traceFrames: [],
+  nodeGraphFrames: [],
+  tableFrames: [],
+  graphResult: null,
+  logsResult: null,
+  tableResult: null,
```

Comment:
this is needed because of the changes in https://github.com/grafana/grafana/pull/45148/files#diff-70e1cc4ad5a5754fe5e6e51db45d1be0501db91555cbfb6db1fc0f80aa328415R200